crates/host/src/wasmbus/mod.rs

```diff
@@ -2820,15 +2832,14 @@ impl Host {
         .await
         .context("failed to uninstantiate running actor")?;
 
-        let new_pool = ActorInstancePool::new(new_actor.clone(), Some(count));
```

Comment:
This change does appear like it would remove the upper limit on instances, but do note that it never actually behaved that way - I was not aware this was introduced and is not how it was designed to be used. See https://github.com/wasmCloud/wasmCloud/pull/574 for more context

This simply defines the maximum amount of instances pooled, which has no effect on the amount of instances that could be active at any point of time